src/components/Tree/Node.tsx

```diff
@@ -25,67 +27,89 @@ type NodeProps = {
 };
 
 export const TreeNode = ({
-    node: { id, value, name, label, icon, nodes },
+    node: { id, value, name, label, icon, nodes, actions },
     strong = false,
     activeNodeId = null,
     onClick,
     parentIds = [],
 }: NodeProps): ReactElement<NodeProps> => {
     const [showNodes, setShowNodes] = useState(false);
+    const [isHovered, setIsHovered] = useState(false);
     const selected = id === activeNodeId;
 
     /* eslint-disable jsx-a11y/anchor-is-valid,jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions,jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */
     return (
         <li data-test-id="node">
-            <a
-                data-test-id="node-link"
+            <div
                 className={merge([
-                    "tw-flex tw-items-center tw-justify-center tw-py-1 tw-px-2 tw-rounded tw-cursor-pointer tw-no-underline",
+                    " tw-flex tw-py-1 tw-px-2 tw-rounded tw-no-underline tw-leading-6",
                     strong && "tw-font-bold",
                     value && !selected && "hover:tw-bg-black-5",
                     selected ? "tw-bg-violet-60 tw-text-white" : "tw-text-black",
                     parentIds.length === 1 && "tw-pl-8",
                     parentIds.length > 1 && "tw-pl-16",
                 ])}
-                aria-selected={selected}
-                onClick={() => {
-                    if (!value) {
-                        setShowNodes(!showNodes);
-                        return;
-                    }
-
-                    if (onClick) {
-                        onClick(activeNodeId === id ? null : id);
-                    }
-                }}
+                onMouseEnter={() => setIsHovered(true)}
```

Comment:
()=> setIsHoverd(true) should be move outside of render

Reply:
Done